core/domain/exp_domain.py

```diff
@@ -2807,7 +2807,7 @@ def get_trainable_states_dict(self, old_states, exp_versions_diff):
         return trainable_states_dict
 
     @classmethod
-    def _convert_states_v0_dict_to_v1_dict(cls, states_dict):
+    def _convert_states_v0_dict_to_v1_dict(cls, states_dict, exp_id):
```

Comment:
Hi @vojtechjelinek , @seanlip  It gives a linting error (unused argument) but we need the exp_id in the convert_states_v23_dict_to_v24_dict function because we need to get the image from the GCS and for that we need to know the exploration id.

Reply:
I think it will ignore it if you prefix the varname with `unused_`.

Reply:
Okay, Thanks